### ISSUE TITLE
Pre-Defined alpha channel

### ISSUE THREAD

**Issue by Bayerberg** [taufik-nurrohman/color-picker#43]
Having issues with getting predefined alpha channel. I'm pulling rgba value from one of the values from  an array and although it shows up for a moment it gets replaced by rgb(255, 0, 0). 

~~~ .php
      $html .= '<script>';
      $html .= '
      var picker = document.querySelectorAll(\'.picker\'),
          alpha = document.createElement(\'input\');

          picker_0 = new CP(picker[0]),
          picker_1 = new CP(picker[1]),
          picker_2 = new CP(picker[2]),
          picker_3 = new CP(picker[3]);
          picker_4 = new CP(picker[4]);

          alpha.type = \'range\';
          alpha.min = 0;
          alpha.max = 1;
          alpha.step = .01;
          alpha.value = 1;
          alpha.onchange = change;
          alpha.oninput = change;
          picker_4.self.appendChild(alpha);

          function change() {
              var v = CP._HSV2RGB(picker_4.set());
              v = this.value == 1 ? \'rgb(\' + v.join(\', \') + \')\' : \'rgba(\' + v.join(\', \') + \', \' + this.value.replace(/^0\./, \'.\') + \')\';
              picker_4.source.value = v;
          }

          picker_0.on("change", function(color) {
            this.source.value = \'#\' + color;
          });
          picker_1.on("change", function(color) {
            this.source.value = \'#\' + color;
          });
          picker_2.on("change", function(color) {
            this.source.value = \'#\' + color;
          });
          picker_3.on("change", function(color) {
            this.source.value = \'#\' + color;
          });
          picker_4.on("change", function(color) {
            this.source.value = alpha.value == 1 ? \'rgb(\' + CP.HEX2RGB(color).join(\', \') + \')\' : \'rgba(\' + CP.HEX2RGB(color).join(\', \') + \', \' + alpha.value.replace(/^0\./, \'.\') + \')\';
          });
      ';
      $html .='</script>';
~~~

whats the issue?

**Comment by tovic** [comment]
This plugin does not have any support for RGBA color so when it tries to parse RGBA color, it will fall back to the default color which is red.

**Comment by taufik-nurrohman** [comment]
Should be fixed by `v2.0.0`.
